addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3338,7 +3338,231 @@ msgctxt "#802"
 msgid "%s of %s available"
 msgstr ""
 
-#empty strings from id 803 to 849
+#. Label of spinner control for choosing the timer type (one time, repeating, ...)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#803"
+msgid "Timer type:"
```

Comment:
I sort of doubt this label is still used, but it ofc would have to be researched. Also, ideally we'd get rid of the colones all over the place, but that's a bigger change.